### ISSUE TITLE
iOS background refresh

### ISSUE THREAD

**Issue by SqueezedLight** [zo0r/react-native-push-notification#1809]
# Question

TLDR: Should we mention `Settings -> General -> Background Refresh` in the docs (iOS related)?

Hi guys,
i struggled a whole day to trigger `onNotification` when the app is in background (not killed). Foreground always worked, but background didn't. There are A LOT of tickets for this topic. For both platforms iOS and Android. But i couldn't find anything about the settings that solved my problem. I had deactivated `Settings -> General -> Background Refresh` on my iPhone. Since i activated that setting `onNotification` is fired as expected.

Most suggestions link to the [troubleshooting](https://github.com/zo0r/react-native-push-notification/blob/master/trouble-shooting.md) article or to the part in the docs where it's mentioned that you shouldn't call `.configure()` [within a component](https://github.com/zo0r/react-native-push-notification#usage). I think we should also mention this iOS setting - may save lives :D

But then i think how do others accomplish this? I mean have a look a signal (the messenger app) or others. They seem to have a way to act on push notifications no matter if background refresh is on or off. I'm i missing something here?

Thanks for your help!

**Comment by Dallas62** [comment]
Hi @SqueezedLight 

Feel free to provide a PR with needed changes, you can also suggest this change on the iOS repository. https://github.com/react-native-push-notification-ios/push-notification-ios

Regards,
